pages/02.general/02.how-to-create-your-own-ad-filters/docs.ru.md

```diff
@@ -445,8 +446,20 @@ domain.com###banner
 
 Модификаторы, описанные в этом разделе, полностью меняют поведение базовых правил, к которым эти модификаторы применены.
 
+<a id="badfilter-modifier"></a>
+##### **`badfilter`**
+
+Правила, содержащие модификатор `badfilter`, отключают базовые правила, на которые они ссылаются. Это означает, что текст отключенного правила должен соответствовать тексту `badfilter`-правила (за исключением самого модификатора `badfilter`).
+
+###### Примеры `badfilter`
+
+* `||example.com$badfilter` отключает `||example.com`
```

Comment:
To make it compatible with uBO we'd better place `badfilter` in the end of the rule text

badfilter,image -> image,badfilter
badfilter,domain=domain.com -> domain=domain.com,badfilter